src/dtoannotations/AdulthoodValidationProcess.java

```diff
@@ -0,0 +1,29 @@
+package dtoannotations;
+
+import java.lang.reflect.Field;
+import java.time.LocalDate;
+
+public class AdulthoodValidationProcess {
+
+    public String ValidateAdulthoodAnnotation (Object object) throws IllegalAccessException {
+        String text = "";
+        Field[] declaredFields = object.getClass().getDeclaredFields();
+        for (Field declaredField : declaredFields) {
+            declaredField.setAccessible(true);
+            Object fieldValue = declaredField.get(object);
+            if (fieldValue != null) {
+                if (declaredField.isAnnotationPresent(Adulthood.class)) {
```

Comment:
Cause you are going to use **cast** you have to move this **IF statement** upper ( under 12th row )